pkg/v1/providers/ytt/02_addons/avi/ako_operator_secret.yaml

```diff
@@ -29,13 +29,20 @@ akoOperator:
     #@ end
     avi_data_network: #@ data.values.AVI_DATA_NETWORK
     avi_data_network_cidr: #@ data.values.AVI_DATA_NETWORK_CIDR
+
     #@ if data.values.AVI_CONTROL_PLANE_NETWORK and data.values.AVI_CONTROL_PLANE_NETWORK_CIDR:
     avi_control_plane_network: #@ data.values.AVI_CONTROL_PLANE_NETWORK
     avi_control_plane_network_cidr: #@ data.values.AVI_CONTROL_PLANE_NETWORK_CIDR
     #@ else:
+    #@ if data.values.AVI_MANAGEMENT_CLUSTER_VIP_NETWORK_NAME and data.values.AVI_MANAGEMENT_CLUSTER_VIP_NETWORK_CIDR:
+    avi_control_plane_network: #@ data.values.AVI_MANAGEMENT_CLUSTER_VIP_NETWORK_NAME
```

Comment:
My understanding: In this if statement, when control plane network is NOT specified but MC VIP is specified, we should use the MC VIP as the MC's control plane network. Instead of following other WC's data network?

Reply:
is this statement correct?

Reply:
Yes, that's the behavior we have in `v.0.10.0`, need to stick on that for users who are using that until we get VIP network separation in the future.